src/Libraries/CoreNodes/List.cs

```diff
@@ -782,6 +782,38 @@ public static IList Combinations(IList list, int length, bool replace = false)
                     .ToList();
         }
 
+        /// <summary>
+        ///     Return the first index of item in the given list. 
+        ///     Return -1 if not found.
+        /// </summary>
+        /// <param name="list">List to search in</param>
+        /// <param name="item">Item to look for</param>
+        /// <returns></returns>
```

Comment:
Thanks for documenting. Since the documentation is public facing, let's make it slightly more complete when the user cannot see this implementation:

```csharp
/// <summary>
///     Given an item, returns the zero-based index of its first occurrence 
///     in the list. If the item cannot be found in the list, -1 is returned.
/// </summary>
/// <param name="list">
///     List to search in. If this argument is null, -1 is returned.
/// </param>
/// <param name="item">Item to look for.</param>
/// <returns>Zero-based index of the item in the list, or -1 if it is not found.
/// </returns>
```